src/sql/workbench/browser/modelComponents/text.component.ts

```diff
@@ -22,24 +22,40 @@ import { errorForeground } from 'vs/platform/theme/common/colorRegistry';
 
 export enum TextType {
 	Normal = 'Normal',
-	Error = 'Error'
+	Error = 'Error',
+	UnorderedList = 'UnorderedList',
+	OrderedList = 'OrderedList'
 }
 
 const errorTextClass = 'error-text';
 
 @Component({
 	selector: 'modelview-text',
 	template: `
-	<div *ngIf="showDiv;else noDiv" style="display:flex;flex-flow:row;align-items:center;" [style.width]="getWidth()" [style.height]="getHeight()">
-		<p [title]="title" [ngStyle]="this.CSSStyles" [attr.role]="ariaRole" [attr.aria-hidden]="ariaHidden"></p>
-		<div #textContainer id="textContainer"></div>
-		<span *ngIf="requiredIndicator" style="color:red;margin-left:5px;">*</span>
-		<div *ngIf="description" tabindex="0" class="modelview-text-tooltip" [attr.aria-label]="description" role="img">
-			<div class="modelview-text-tooltip-content" [innerHTML]="description"></div>
+	<div *ngIf="showList;else noList" [style.display]="display" [style.width]="getWidth()" [style.height]="getHeight()" [title]="title" [attr.role]="ariaRole" [attr.aria-hidden]="ariaHidden" [ngStyle]="this.CSSStyles">
+		<div *ngIf="isUnOrderedList;else orderedlist">
+			<ul style="padding-left:0px">
+				<li *ngFor="let v of value">{{v}}</li>
```

Comment:
I believe Angular will automatically sanitize the string here, could you please try to use a list of valid html strings and make sure they are being treated as strings not html.

Reply:
```
<modelview-text style="overflow: hidden; position: relative;">
	<!--bindings={
  "ng-reflect-ng-if": "true",
  "ng-reflect-ng-if-else": "[object Object]"
}--><div ng-reflect-ng-style="[object Object]" title="" role="" aria-hidden="false" style="padding-left: 30px; margin-bottom: 5px; margin-top: 10px;">
		<!--bindings={
  "ng-reflect-ng-if": "true",
  "ng-reflect-ng-if-else": "[object Object]"
}--><div>
			<ul style="padding-left:0px">
				<!--bindings={
  "ng-reflect-ng-for-of": "Azure account details,Azure SQ"
}--><li>Azure account details</li><li>Azure SQL Managed Instance or SQL Server on Azure Virtual Machine</li><li>Backup location details</li><li>&lt;span style="color:red;"&gt; fadsfasd &lt;/span&gt;</li>
			</ul>
		</div>
		<!---->
	</div>
	<!----></modelview-text>
```